posthog/temporal/batch_exports/redshift_batch_export.py

```diff
@@ -302,7 +304,14 @@ async def insert_records_to_redshift(
         table=table_identifier,
         fields=sql.SQL(", ").join(map(sql.Identifier, columns)),
     )
-    template = sql.SQL("({})").format(sql.SQL(", ").join(map(sql.Placeholder, columns)))
+    placeholders: list[sql.Composable] = []
+    for column in columns:
+        if use_super is True and known_super_columns is not None and column in known_super_columns:
+            placeholders.append(sql.SQL("JSON_PARSE({placeholder})").format(placeholder=sql.Placeholder(column)))
+        else:
+            placeholders.append(sql.Placeholder(column))
+
+    template = sql.SQL("({})").format(sql.SQL(", ").join(placeholders))
```

Comment:
TODO: `INSERT` queries like this one are not efficient in Redshift. Unfortunately, Redshift's `COPY` only supports files from S3, not local file system, so we would require additional permissions from users. There's an item to switch this to `COPY` from S3.

Reply:
Redshift does support remote files via an SSH connection, so that could be an alternative to S3 that doesn't require additional setup from the user's side.